crates/stackable-operator/src/builder/event.rs

```diff
@@ -13,11 +15,11 @@ pub enum EventType {
     Warning,
 }
 
-impl ToString for EventType {
-    fn to_string(&self) -> String {
+impl fmt::Display for EventType {
```

Comment:
nit (optional): We could use [`strum`](https://docs.rs/strum/latest/strum/derive.Display.html) for that

Reply:
Oh right. I can adjust that.

Reply:
Changed in c14affa